acceptance/config_test.go

```diff
@@ -70,6 +70,9 @@ type ServerStub struct {
 
 	// The response body to return.
 	Response testserver.Response
+
+	// Artificial delay in seconds to simulate slow responses.
+	DelaySeconds float64
```

Comment:
This should be `*float64`, so that explicitly setting it to 0 works (otherwise mergo treats it as null value to be skipped).